main.js

```diff
@@ -457,60 +455,50 @@ function getRowOfLastVisibleCell() {
     }
 }
 
+function getCurrentConfigurations() {
+    var configurations = {};
+    currentWidgets.forEach(function(widget, index) {
+        let configName = widget.config;
+        let configText = getItemConfigText(index);
+        let configJSONObject = parseJSONString(configText, index);
+
+        configurations[configName] = configJSONObject;
+    });
+    return configurations;
+}
+
 // Build the GridData object and convert it to a JSON string
 function generateJSON() {
-    grid.synchronize();//reflect any order changes
+    grid.synchronize(); //reflect any order changes
+
+    // Not really sure how else to do this.
+    // I grab the "top" and "left" coordinates of the element and divide it by the base width/height (+4px of padding). Round it since it sometimes is slightly off and columns won't be non-integers 
+    grid._items.forEach(function(item, index) {
+        let widgetId = gridItemDict[item._id];
+        if (currentWidgets[widgetId]) {
+            console.log("Left: " + item._left);
```

Comment:
Should these `console.log` statements still be here?

Reply:
Nope!